apis/nucleus/src/components/listbox/interactions/listbox-keyboard-navigation.js

```diff
@@ -234,11 +234,18 @@ export function getListboxInlineKeyboardNavigation({
     }
 
     const container = event.currentTarget.closest('.listbox-container');
+    const inSelection = isModal();
 
     switch (keyCode) {
       case KEYS.TAB:
-        if (shiftKey) {
-          focusRow(container) || focusSearch(container);
+        if (inSelection) {
```

Comment:
I think
```
if (!inSelection) {
   return undefined;
}
```
would be a closer match to how the old filter pane work.
(but maybe that has to may problem from other differences?)

Reply:
Yeah, that would be closer to how the existing Filter pane works. I discussed it with @pernyl quickly and we think that – for the case when there are no selections – tabbing (without shift) should do the opposite of tabbing with shift. In other words:
- Shift + Tab: Go to parent cell
- Tab: Go into listbox (search field or row)

I think that's an improvement compared to the existing Filter pane.